ui/src/pages/repos/index.tsx

```diff
@@ -103,7 +103,7 @@ function RepoLine({ repo, deletable, sharable }) {
       sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
     >
       <TableCell align="center">
-        <Link component={ReactLink} to={`/repo/${repo.id}`}>
+        <Link component={ReactLink} to={`/repo/${repo.id}?name=${repo.name}`}>
```

Comment:
We don't want the repo name to be in the URL, it's duplicate information. Although it works for this link, a shareable link like `https://app.codepod.io/repo/repo_HwYAxxbF` won't work.  There got to be another way to pass the name to the header.

For example, we could move the `<Header>` component down into the `<Repo>` component.

Reply:
it's normally store path breadcrums in the location
![image](https://user-images.githubusercontent.com/20850940/204813985-535ee003-4d9c-4c87-8a97-4cd1122faa67.png)
deepnote parse location to show the path

Reply:
@nibilin33 
1. Please do not change the design of CodePod. If you have to, discuss with me and/or Hebi before you change. We ask you to only change the UI. If you keep doing so, we will not be able to work with you. 
2. Your change is passing an argument `name` what will be not handled by CodePod server for now. Such a change could break other parts of the system. Hence, please stop making unilateral changes. 
3. You referred to DeepNote's practice. But we have a pending discussion on who to generate URLs. Actually I pointed out that DeepNote's design is not the smartest. https://github.com/codepod-io/codepod/issues/91 

cc @lihebi

Reply:
?? the situation is that shareable link do not contain name info so once the user use shareable link, the header path will be not correctly but user can still enter the system.

Reply:
> shareable link do not contain name info

The link already has the repo ID; including the repo name would be redundant. Instead, the repo name should be loaded from the server and the header can then be set.

> user can still enter the system

What do you mean?

Reply:
> > shareable link do not contain name info
> 
> The link already has the repo ID; including the repo name would be redundant. Instead, the repo name should be loaded from the server and the header can then be set.

Agree. Querying the database to get the repo name from repo id is the correct way. Repo ID is permanent and unique. Repo name can change. 

> 
> > user can still enter the system
> 
> What do you mean?

I think she means that users can still view and edit a shareable repo but the repo name will be displayed incorrectly.

Reply:
I just explain this change didn't affect server just ui display.

Reply:
that's it. nice talking. i'm done.